translations/bg_BG.json

```diff
@@ -613,6 +613,7 @@
     "virtAsstCGMBattOne": "Your CGM battery was %1 volts as of %2.",
     "virtAsstCGMBattTwo": "Your CGM battery levels were %1 volts and %2 volts as of %3.",
     "virtAsstDelta": "Your delta was %1 between %2 and %3.",
+    "virtAsstDeltaEstimated": "Your estimated delta was %1 between %2 and %3.",
```

Comment:
Can you remove this file from the PR? The language files are changed through Crowdin and pull requests are are not expected to change these files.

Reply:
If you want a new key in the system, add that to language/en/en.json

Reply:
Got it. These have been removed and merge conflicts resolved.